src/SpecFlow.NetCore/Fixer.cs

```diff
@@ -56,22 +57,35 @@ private void DeleteFakeCsProj(FileInfo fakeCsproj)
 			fakeCsproj.Delete();
 		}
 
-		private void FixXunit(DirectoryInfo directory)
+		private void FixTests(DirectoryInfo directory)
 		{
 			WriteLine("Fixing SpecFlow generated files for xUnit 2");
 
 			var glueFiles = directory.GetFiles("*.feature.cs", SearchOption.AllDirectories);
 
 			foreach (var glueFile in glueFiles)
 			{
-				WriteLine("Fixed: " + glueFile.FullName);
+				WriteLine("Fixing: " + glueFile.FullName);
 				var content = File.ReadAllText(glueFile.FullName);
-				content = content.Replace(" : Xunit.IUseFixture<", " : Xunit.IClassFixture<");
-				content = content.Replace("[Xunit.Extensions", "[Xunit");
+				content = FixXUnit(content);
+				content = FixMsTest(content);

```

Comment:
Yeah the mstest one is a regex so might be appropriate. I'll let you decide on argument/guess first and I'll fix this once we have a decision there.